microsoft-edge/webdriver-chromium/ie-mode.md

```diff
@@ -0,0 +1,376 @@
+
+Organizations with business-critical legacy websites or apps may need to test their content in IE mode in Microsoft Edge.  The following sections describe how to get started with Internet Explorer Driver (IEDriver) to automate IE mode in Microsoft Edge.
+
+
+<!-- ====================================================================== -->
+## Download Internet Explorer Driver (IEDriver)
+
+To begin automating tests in IE mode in Microsoft Edge, [download IEDriver](https://www.selenium.dev/downloads/).  Make sure that the version of IEDriver that you download is `4.0.0.0` or greater.
+
+:::image type="content" source="./media/iedriver-install.msft.png" alt-text="The IEDriver section of Downloads page for Selenium." lightbox="./media/iedriver-install.msft.png":::
+
+
+<!-- ====================================================================== -->
+## Automate IE mode in Microsoft Edge
+
+The following sections walk you through using Selenium to automate IE mode in Microsoft Edge.
+
+> [!NOTE]
+> This article provides instructions for using the Selenium framework, but you can use any library, framework, and programming language that supports WebDriver.  To accomplish the same tasks using another framework, consult the documentation for your framework of choice.
+
+To launch Microsoft Edge in IE mode with IEDriver:
+
+1.  Create an `InternetExplorerDriverService`.
+
+1.  Define `InternetExplorerOptions` with additional capabilities that point to the Microsoft Edge browser.
+
+1.  Start an instance of `InternetExplorerDriver` and pass it `InternetExplorerDriverService` and `InternetExplorerOptions`.  IEDriver launches Microsoft Edge and then loads your web content in IE mode.
+
+The next section shows the complete sample, and then the subsequent sections focus on each of the main steps that are listed above.
+
+
+<!-- ====================================================================== -->
+## The complete sample
+
+The following sample launches Microsoft Edge in IE mode, navigates to [bing.com](https://www.bing.com/) and searches for "WebDriver".
+
+### [C#](#tab/c-sharp/)
+
+```csharp
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.IE;
+
+namespace Selenium_Web
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //change the path accordingly
+            var dir = @"C:\<path to IEDriver download>";
+            var driverexe = "IEDriverServer.exe";
+
+            var ieService = 
+                 InternetExplorerDriverService.CreateDefaultService(dir, driverexe);
+            var ieOptions = new InternetExplorerOptions { IgnoreZoomLevel = true };
+            ieOptions.AttachToEdgeChrome = true;
+            //change the path accordingly
+            ieOptions.EdgeExecutablePath = "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe";
```

Comment:
Global: Consider wrapping code blocks at column 85 per https://docs.microsoft.com/en-us/contribute/code-in-docs#horizontal-scroll-bars